src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -66,7 +67,22 @@ export default {
         this.targetIsNotALane() &&
         this.targetIsInSamePool() &&
         this.targetIsNotSource() &&
-        this.validateOutgoing();
+        this.validateOutgoing() &&
+        this.targetInclusiveGateway();
+    },
+    targetInclusiveGateway() {
```

Comment:
I'm not sure this is the best place for this function. This function is called from `isValidTarget`, but it doesn't actually validate anything, it modifies the target if it's an inclusive gateway.

If possible, I think this logic should be moved to the actual inclusive gateway component.